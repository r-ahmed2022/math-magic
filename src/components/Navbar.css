.navbar {
  display: flex;
  justify-content: space-between;
  align-items: center;
  width: 100%;
  height: 80px;
  border: 2px solid #000;
  margin: 0 auto;
  padding-right: 30px;
  font-family: 'Nanum Gothic Coding', monospace;
  font-weight: 700;
  font-size: 1.2rem;
}

.navlinks {
  display: flex;
  list-style: none;
}

.link {
  margin: 5px;
  border-right: 2px solid #000;
  padding: 0 10px;
}


.header-title {
  margin-left: 20px;
  font-size: 2rem;
}
