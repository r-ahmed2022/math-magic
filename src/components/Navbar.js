import React from 'react';
import { Link } from 'react-router-dom';
import './Navbar.css';

export default function Navbar() {
  return (
    <nav className="navbar">
      <h1 className="header-title"> Math Magicians</h1>
      <ul className="navlinks">
        <li className="link"><Link to="/" className="">Home</Link></li>
        <li className="link"><Link to="/calculator" className="">Calculator</Link></li>
        <li className="link last"><Link to="/quote" className="">Quote</Link></li>
      </ul>
    </nav>
  );
}
