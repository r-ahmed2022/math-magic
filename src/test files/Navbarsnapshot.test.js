import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Navbar from '../components/Navbar';

it('Navbar renders correctly', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <Navbar />
    </MemoryRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
