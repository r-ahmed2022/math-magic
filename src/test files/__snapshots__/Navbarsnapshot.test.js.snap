// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<nav
  className="navbar"
>
  <h1
    className="header-title"
  >
     Math Magicians
  </h1>
  <ul
    className="navlinks"
  >
    <li
      className="link"
    >
      <a
        className=""
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="link"
    >
      <a
        className=""
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li
      className="link last"
    >
      <a
        className=""
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
